README_.google

```diff
@@ -1,5 +1,5 @@
 URL: http://codemirror.net/
-Version: 5.58.0
+Version: 5.59.0
```

Comment:
When is 5.59.0 going to be released? [CodeMirror.net](https://codemirror.net/) shows the current version as 5.58.1.

Reply:
That's right, I updated it back to 5.58.0. I'm not updating the codemirror version.